gallery/src/components/NxFilterInput/NxFilterInputPage.tsx

```diff
@@ -25,7 +25,13 @@ const NxFilterInputPage = () =>
   <>
     <GalleryDescriptionTile>
       <NxP>
-        An input to be used for filtering purposes
+        An input to be used for filtering purposes. After a user has text within the input field, a
+        clear filter button appears within the component and is only visible/usable when there is text within
+        the input field.
+      </NxP>
+      <NxP>
+        Note: This button is for mouse use only and is therefore not focusable but still hoverable and
```

Comment:
IMO this should be in the same paragraph as the previous sentence

Reply:
Updated here be92dbb5730ca5e08c7792134dfb29890331f319. Thanks.